setup.py

```diff
@@ -4,11 +4,14 @@
 from setuptools import setup, find_packages
 from os         import path
 
+import cvmfs
+
+
 readme_path = path.join(path.dirname(__file__), 'README')
 
 setup(
-  name="python-cvmfsutils",
-  version="0.2.0",
+  name=cvmfs.package_name,
+  version=cvmfs.version,
```

Comment:
I was thinking of a sub-package in `cvmfs` but that doesn't work either - just tried it. If you do the following in `setup.py` it still runs `cvmfs/__init__.py` and not only `cvmfs/__meta_info__/__init__.py`.

```python
import cvmfs.__meta_info__
```

I guess we are stuck with the non-DRY version then. :-(